tests/snappi/ecn/files/helper.py

```diff
@@ -296,6 +297,10 @@ def __run_traffic(api,
         N/A
     """
     api.set_config(config)
+
+    # """Wait for Arp"""
```

Comment:
Why do you use both # and """?

Reply:
Sorry..didn't notice the typo. Updated the comment to log info